app/Domain/UserVacationStatsRetriever.php

```diff
@@ -103,6 +103,15 @@ public function getVacationDaysLimit(User $user, YearPeriod $yearPeriod): int
         return $limit ?? 0;
     }
 
+    public function hasVacationDaysLimit(User $user, YearPeriod $yearPeriod): bool
```

Comment:
The method above `getVacationDaysLimit` looks almost exactly the same as this method. The only difference is return type. We shouldn't duplicate code, but think how to use existing code.